math/minuit2/inc/Minuit2/MnUserTransformation.h

```diff
@@ -91,6 +92,8 @@ class MnUserTransformation {
 
    // Index = internal Parameter
    double DInt2Ext(unsigned int, double) const;
+   double D2Int2Ext(unsigned int, double) const;
+   double GStepInt2Ext(unsigned int, double) const;
```

Comment:
I think the GStepInt2Ext function is not needed and could be removed.
Se comment below

Reply:
Indeed :)